ui/src/pages/Dashboard.vue

```diff
@@ -62,13 +63,19 @@
   } from '@prefecthq/prefect-ui-library'
   import { useSubscription } from '@prefecthq/vue-compositions'
   import { computed, provide } from 'vue'
-
+  import { usePrefectApi } from '@/compositions/usePrefectApi'
 
   provide(subscriptionIntervalKey, {
     interval: dateFunctions.secondsToMilliseconds(30),
   })
 
   const api = useWorkspaceApi()
+  const prefectApi = usePrefectApi()
+  const serverSettings = await prefectApi.admin.getSettings()
+  const showPromotionalContent = computed(() => serverSettings.server.ui.show_promotional_content)
+
+  // Cache to localStorage for use in error toasts
+  localStorage.setItem('prefect-show-promotional-content', String(showPromotionalContent.value))
```

Comment:
was thinking this is necessary bc oftentimes this toast is meant to show when you can't connect to the server, which means we can't read the settings

lmk if a different approach would be better